SiraUtil/Objects/InternalRedecorator.cs

```diff
@@ -83,7 +95,7 @@ internal static void Redecorate(ref List<CodeInstruction> codes)
 
             for (int i = 0; i < codes.Count - 1; i++)
             {
-                if (codes[i].opcode == OpCodes.Ldfld && (codes[i + 1].Calls(_newPrefabMethod) || (codes.Count > i + 4 && codes[i + 4].Calls(_newPrefabMethod)))) // uhhh for teranary operators :PogOh:
+                if (codes[i].opcode == OpCodes.Ldfld && (codes[i + 1].Calls(_newPrefabMethod) || (codes[i + 1].opcode == OpCodes.Callvirt && ((MethodInfo)codes[i + 1].operand).Name == "ByNewContextPrefab") || (codes.Count > i + 4 && codes[i + 4].Calls(_newPrefabMethod)))) // uhhh for teranary operators :PogOh:
```

Comment:
hmm, do you think you can convert that Name check to a .Calls check?

Reply:
no, you cannot convert the name check to a .Calls check because the class is generic

Reply:
Can you `nameof` it or make it a constant?

Reply:
done